tests/testthat/test_updateMeta.R

```diff
@@ -21,11 +21,13 @@ test_that("Remove rows meta helper", {
 })
 
 test_that("Add rows to meta helper", {
-  expect_message(add_rows_meta(df1$labels, newDat), "Adding meta data for the following variables: v3")
-  expect_equal(suppressMessages(add_rows_meta(df1$labels, newDat)), import_DF(newDat[, "v3", drop = F]))
+  expect_message(out <- add_rows_meta(df1$labels, newDat, checkVarNames = FALSE),
+                 "Adding meta data for the following variables: v3")

```

Comment:
LGTM (however, I could not see the point in setting the `checkVarNames` arg to `FALSE`, here and in the function bodies?  🤔- the test also passes for `TRUE` - this might be due to my lacking overview of `updateMeta()`, sorry 🙈)

Reply:
Indeed, for the specific example `checkVarNames` does not make a difference, as no invalid variable names are used. If invalid variable names are used, the argument becomes meaningful. It's functionality is, however, tested in the global `updateMeta()` test.